service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/PatientMapper.java

```diff
@@ -0,0 +1,85 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import lombok.RequiredArgsConstructor;
+import org.hl7.fhir.dstu3.model.Enumerations;
+import org.hl7.fhir.dstu3.model.Patient;
+import uk.nhs.adaptors.oneoneone.cda.report.enums.MaritalStatus;
+import org.springframework.stereotype.Component;
+import uk.nhs.adaptors.oneoneone.cda.report.service.OrganizationService;
+import uk.nhs.connect.iucds.cda.ucr.*;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.stream.Stream;
+
```

Comment:
@babasholaking-kainos  is also creating patient mapper, for he does the encounter.subject. @Broom94, please contact with him and discuss this mapper to not have any serious conflicts later.